feat(compose): passthrough bindings + support containerless (#1792)
aurelia/aurelia: docs/user-docs/getting-to-know-aurelia/dynamic-composition.md

<au-compose view-model.ref="myCompose"></au-compose>
```

> This will works as if it was a `view-model.ref` binding on a normal custom element

Reviewer: ```suggestion
This will work as though it were a `view-model.ref` binding on a standard custom element.
```

## Passing props through

> The `<au-compose>` will pass all the bindings, except the ones that target its bindable properties (`model`/`component`/`template`) declared on it to the composed view model, if it's a custom element.

Reviewer: ```suggestion
The `<au-compose>` will pass all bindings, except those targeting its bindable properties (`model`/`component`/`template`) declared on it, to the composed view model, assuming it is a custom element.
```

The above value converter will load the URL and return the text response. For view models, something similar can be achieved where an object or class can be returned.

> 3. In Aurelia 2, all bindings are passed through to the underlying custom element
> composition, so `view-model.ref` no longer means getting a reference to the composer, but the composed view model instead.
> 

Reviewer: ```suggestion
3. In Aurelia 2, all bindings are transferred to the underlying custom element composition. Therefore, `view-model.ref` no longer signifies obtaining a reference to the composer, but rather to the composed view model.
```
